xrpl/models/requests/request.py

```diff
@@ -147,17 +146,20 @@ def get_method(cls: Type[Request], method: str) -> Type[Request]:
             The request class with the given name. If the request doesn't exist, then
             it will return a `GenericRequest`.
         """
-        import xrpl.models.requests as request_models
-
-        request_methods: Dict[str, Type[Request]] = {}
-        for r in RequestMethod:
-            method_name = _method_to_class_name(r.name)
-            request_methods[r.value] = getattr(request_models, method_name)
-
-        if method in request_methods:
-            return request_methods[method]
-
-        return request_models.GenericRequest
+        # special case for NoRippleCheck and NFT methods
+        if method == RequestMethod.NO_RIPPLE_CHECK:
+            return xrpl.models.requests.NoRippleCheck
+        if method == RequestMethod.ACCOUNT_NFTS:
+            return xrpl.models.requests.AccountNFTs
+        if method == RequestMethod.NFT_BUY_OFFERS:
+            return xrpl.models.requests.NFTBuyOffers
+        if method == RequestMethod.NFT_SELL_OFFERS:
+            return xrpl.models.requests.NFTSellOffers
```

Comment:
Not sure if we can do this in Python but can we reduce this to `return xrpl.models.requests[RequestMethod.METHOD_NAME]`

Reply:
Oh wait, there's more logic after this conditional check. Nevermind :)